app/src/main/groovy/ua/net/nlp/api/BatchController.groovy

```diff
@@ -39,14 +50,21 @@ class BatchController extends BaseController {
     @PostMapping(path="/batch")
     def batch(
         @io.swagger.v3.oas.annotations.parameters.RequestBody(
-            description='Body text; e.g<br>{"text": "Сьогодні y продажi «ХХІ століття»."}', 
+            description='Body text; e.g<br>{"texts": ["Сьогодні y продажi «ХХІ століття»."]}', 
             required = true)
         @RequestBody BatchRequest request) {
         
         validateRequest(request)
 
         try {
-            batchService.batch(request.text)
+            // To Andriy. Can I use parallel here? groovy gave me the error when compiling.
+            // Maybe it is related to the old version of it?
+            // Groovy Version: 4.0.5 JVM: 17.0.4.1 Vendor: Homebrew OS: Mac OS X
+            // It's not a mandatory option, so if it's hard to implement, I'd rather skip this one
+            // return request.texts.parallel().collect { text ->
+            return request.texts.collect { text ->
```

Comment:
You can try Java parallel stream + (simplified) Groovy collect:
```
request.tests.parallelStream().collect{ 
  batchService.batch(text)
}
```

Reply:
lang-uk/nlp_uk_api/app/src/main/groovy/ua/net/nlp/api/BatchController.groovy: 65: [Static type checking] - Cannot find matching method java.util.List#parallelStream(groovy.lang.Closure). Please check if the declared type is correct and if the method exists.
 @ line 65, column 20.
               return request.texts.parallelStream() { text ->

Reply:
`texts.parallelStream().collect{ ... }`

Reply:
/lang-uk/nlp_uk_api/app/src/main/groovy/ua/net/nlp/api/BatchController.groovy: 66: [Static type checking] - Cannot find matching method ua.net.nlp.api.services.BatchService#batch(java.lang.Object). Please check if the declared type is correct and if the method exists.
 @ line 66, column 17.
                   batchService.batch(text)
                   ^

1 error


```

Reply:
`parallelStream().collect{ String text ->`